### DISCUSSION TITLE
欢迎来到shokaX讨论区

### DISCUSSION THREAD

**Discussion by zkz098** [theme-shoka-x/hexo-theme-shokaX#9] [Announcements]
## 欢迎👋
此处是shokaX的交流讨论区,您可以:
- 提出您想问的问题
- 分享您对于shokaX或社区的建议与想法
- 参与到讨论中来
- 讨论一切与shokaX相关的事
- 讨论一些和网站建设有关的事

**Comment by argvchs** [comment]
https://github.com/zkz098/hexo-theme-shokaX/commit/8475e3c22081cb8426ad995e96b2a6867eb76af6

Shoka 上来就有 Algolia 搜索，二级导航栏，暗黑模式，音乐播放，文章目录，文章置顶，系列文章，代码复制了

ShokaX 甚至还有 ESLint，TypeScript，Install CLI，Issue 模板

您太强了

**Reply by zkz098** [reply]
感谢夸奖,[ParticleX](https://github.com/argvchs/hexo-theme-particlex)的技术跨度比ShokaX大多了,目前为止ShokaX还只能算是shoka的一个魔改版
